packages/components/dialog/src/dialog-content.vue

```diff
@@ -61,5 +61,5 @@ const { focusTrapRef } = inject(FOCUS_TRAP_INJECTION_KEY)!
 const composedDialogRef = composeRefs(focusTrapRef, dialogRef)
 
 const draggable = computed(() => props.draggable)
-useDraggable(dialogRef, headerRef, draggable)
+useDraggable(dialogRef, headerRef, draggable, ['bottom'])
```

Comment:
The fourth parameter here is directly set to `['bottom']`, so the following judgments about `left`, `right` and `top` seem to be useless?

Reply:
By default, all bounds will be bounded, this is just to address issue #9100, #9324, so just let go of the bounds below, of course this is optional